src/functional/resources/repo-config.csv

```diff
@@ -1,5 +1,5 @@
-Repository's Location,Branch,Ignore Glob List,Ignore standalone config
-https://github.com/reposense/testrepo-Alpha.git,master,,
-https://github.com/reposense/testrepo-Beta.git,master,docs**,yes
-https://github.com/reposense/testrepo-Charlie.git,master,,
-https://github.com/reposense/testrepo-Delta.git,master,,
+Repository's Location,Branch,File formats,Ignore Glob List,Ignore standalone config
+https://github.com/reposense/testrepo-Alpha.git,master,java;adoc,,
+https://github.com/reposense/testrepo-Beta.git,master,java;adoc,docs**,yes
```

Comment:
following the functional cli change, let's make all empty, other than beta. Beta should use a diff value from cli arg.